includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -90,6 +101,24 @@ public static function render_outbox_purge_days_field() {
 		) . '</p>';
 	}
 
+	/**
+	 * Render vary header field.
+	 */
+	public static function render_vary_header_field() {
+		$value = \get_option( 'activitypub_vary_header', '0' );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" id="activitypub_vary_header" name="activitypub_vary_header" value="1" <?php checked( '1', $value ); ?> />
+				<?php echo \wp_kses( \__( 'Add the <code>Accept</code> Vary header for <code>application/activity+json</code> requests to the ActivityPub response.', 'activitypub' ), array( 'code' => array() ) ); ?>
```

Comment:
Wording suggestion that might be more user-friendly: "Help prevent incorrect caching of ActivityPub responses"